classes.py

```diff
@@ -0,0 +1,24 @@
+class DNA(str):
```

Comment:
PEP8 - name of class must begin with capital letter, other should be lowercase

Reply:
so named for the habit.
```suggestion
class Dna(str):
```